ios/PurchasesHybridCommon/Info.plist

```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>$(PRODUCT_BUNDLE_PACKAGE_TYPE)</string>
 	<key>CFBundleShortVersionString</key>
-	<string>$(MARKETING_VERSION)</string>
+	<string>1.3.0</string>
```

Comment:
fastlane made this change, is this fine?

Reply:
🤔 did it also update the version in the project file?

Reply:
are you using the latest version of fastlane? was this with xcode 11 or 12? maybe there's some incompatibility with the formats and the project doesn't update?

Reply:
What file is supposed to update, the xcproject? This was both in Xcode 11 and 12

Reply:
I looked at this https://github.com/RevenueCat/purchases-hybrid-common/pull/42/files and no xcproject was updated

Reply:
I see. So there's a Marketing Version in the Build Settings

Reply:
Is it fine if we let Fastlane increment the `CFBundleShortVersionString` like in this commit, and then we add something like the following to increment the `MARKETING_VERSION` in the Build Settings:

```
version=$(sed -n '/MARKETING_VERSION/{s/MARKETING_VERSION = //;s/;//;s/^[[:space:]]*//;p;q;}' {your project name}.xcodeproj/project.pbxproj)a
```

Reply:
🤔 I'm surprised that fastlane bump doesn't do this automatically, it does for purchases-ios. 
also, I just noticed that the fastfile does bump by calling "sh fastlane ios ..." through the console, but it doesn't need to, it can just call the command directly as in https://github.com/RevenueCat/purchases-ios/blob/develop/fastlane/Fastfile#L34 or even call the method `bump` directly

Reply:
Please note that this Fastfile is in the root folder so it has both android and ios platform lanes. That's the only way I could find to specify the platform I wanted to do the bump on

Reply:
Also, the fastlane in iOS doesn't increment the `MARKETING_VERSION`, it also increments `CFBundleShortVersionString` like in this repo https://github.com/RevenueCat/purchases-ios/pull/274/files#diff-d998ce5a98d33edbdd2da993fd55f1db